bedrock/mozorg/redirects.py

```diff
@@ -228,6 +228,10 @@ def decider(request, **kwargs):
     # Bug 1073269 /dnt/ -> /firefox/dnt/
     redirect(r'^dnt/?$', 'firefox.dnt'),
 
+    # bug 1205632
+    redirect('^js/language(?P<path>.*)$',
```

Comment:
I think we should go ahead and move the [other](http://viewvc.svn.mozilla.org/vc/projects/mozilla.com/trunk/.htaccess?view=markup#l753) [redirects](http://viewvc.svn.mozilla.org/vc/projects/mozilla.com/trunk/.htaccess?view=markup#l694) here as well. Those can go above this one and then this one will catch the rest. Redirects are tried in order, and before the other URLs.